docs/src/advanced/symmetries.md

```diff
@@ -51,6 +51,25 @@ one can find a reduced set of ``k``-points
 (the *irreducible ``k``-points*) such that the eigenvectors at the
 reducible ``k``-points can be deduced from those at the irreducible ``k``-points.
 
+### Symmetrization
+Quantities that are calculated by summing over the reducible ``k`` points can be
+calculated by first summing over the irreducible ``k`` points and then symmetrizing.
+In this subsection, we denote by ``S`` the combined rotation and fractional transformation.
+If ``S`` is the symmetry of the system, ``f(Sk) = S(f(k))`` holds, where ``S(f)`` is the
+symmetry transformation of ``f`` which is a linear function and depends on the quantity
```

Comment:
What does it mean for f to be linear in this context? I would just say "consider a k-dependent quantity of interest (energy, density, force...). f is assumed to transform in a particular way under the symmetry: f(S(k)) = S(f(k)) where the action of S on f depends on the particular f."

Reply:
Your writing is clear. What I meant was that S is a linear function.

Reply:
Your writing is clear. What I meant was that S is a linear function.